projects/demo/src/modules/components/mobile-calendar/examples/6/index.less

```
@@ -0,0 +1,3 @@
+.example {
+    height: 35rem;
+}
```

Let's drop redundant nesting

```suggestion
:host {
    display: block;
    height: 35rem;
}
```

\+ delete `<div class="example">` from HTML-file